src/signature.js

```diff
@@ -51,6 +51,15 @@ export class Signature {
   armor(config = defaultConfig) {
     return armor(enums.armor.signature, this.write(), undefined, undefined, undefined, config);
   }
+
+  /**
+   * 
+   * @returns Returns an array of KeyIDs of all of the issuers who created this signature
+   * @returns {Array<KeyID>} The IDs of the issuers
+   */
+  getIssuerIDs() {
```

Comment:
It might be good to be consistent with `Message#getSigningKeyIDs`, and call this function the same thing.